### ISSUE TITLE
Colors for outputter

### ISSUE THREAD

**Issue by sils1297** [coala-analyzer/coala#174]
depends on #168

**Comment by sils1297** [comment]
Okay, we either need to depend on some library (which may complicate installation on several platforms - I'm looking at you, Microsoft) or use those escapes. I read that they work on apples unixoid OS, linux (trivially) and even windows if one activates some ansi.sys thing.

**Comment by sils1297** [comment]
@Makman2 you're working on the windows port, could you do some research along or at least keep the eyes open?

**Comment by Makman2** [comment]
Quick googling there seems to be a console application: ansicon
As far as I understand tt wraps your escape sequences into color API calls so it prints in color.
*ansi.sys* seems to be only the working mechanism in MS-DOS^^

**Comment by sils1297** [comment]
Well MS DOS is more modern than windows 8 in terms of usability ;)

**Comment by sils1297** [comment]
@Makman2 I'll do this for now without worrying about windows and let you do the hard work in the end. If colors aren't possible in the end we can block them.

**Comment by Makman2** [comment]
In first place I would go for blocking them on windows (or just ignore them and print a warning message or whatever). Colors in console is problematic, I think easiest in PowerShell, but also not really cool^^
We can open a separate issue for windows then that we fix someday we want^^

**Comment by sils1297** [comment]
ok, we can do that on ColorPrinter level easily

**Comment by sils1297** [comment]
![screenshot from 2015-04-29 10-38-30](https://cloud.githubusercontent.com/assets/5716520/7387233/df995196-ee5b-11e4-9954-fe3c2d49363d.png)

**Comment by sils1297** [comment]
this is with a dark color scheme from terminal:
![screenshot from 2015-04-29 11-00-11](https://cloud.githubusercontent.com/assets/5716520/7387614/e8e376c0-ee5e-11e4-80d0-19ec97577c3b.png)

**Comment by Makman2** [comment]
Wow this is really cool, especially the code is really better readable :)
